### DISCUSSION TITLE
[Esame 13 febbraio 2024] Esercizio 3

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#448] [Esami M-Z]
![13 feb es.3](https://github.com/user-attachments/assets/eb38f75e-54b6-4724-bc4f-1c88f2d4c41c)

**Comment by niccolo-fato** [comment]
<img width="652" height="1280" alt="image" src="https://github.com/user-attachments/assets/d618c805-aa1e-4b10-a39b-9a29175503b0" />
